tools/c7n_mailer/c7n_mailer/sqs_message_processor.py

```diff
@@ -107,67 +102,118 @@ def process_sqs_messsage(self, message):
         if 'event-owner' in targets:
             targets.remove('event-owner')
             recipient = self.get_aws_username_from_event(data['event'])
-            if recipient is not None:
+            if recipient:
                 targets.add(recipient)
+        if 'ldap_username_tags' in targets:
+            targets.remove('ldap_username_tags')
+            self.send_ldap_username_tags_email(data)
         if not targets:
-            self.logger.info("No intended targets found for message")
+            self.logger.debug("No more intended targets found for message")
             return
-        self.send_message_to_targets(targets, data, data['resources'])
+        self.send_message_to_targets(targets, data)
+
+    def send_ldap_username_tags_email(self, data):
```

Comment:
this is the main function that does an email lookup in LDAP and sends off emails. note you can add multiple tags that you expect to be LDAP usernames. though most folks will likely only use one tag (the auto-tagger tag they use for aws_username)